deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -284,7 +315,7 @@ spec:
                 - /usr/local/bin/ao-logs
                 - /tmp/ansible-operator/runner
                 - stdout
-                image: image-registry.openshift-image-registry.svc:5000/service-telemetry/service-telemetry-operator:latest
+                image: quay.io/infrawatch/service-telemetry-operator:latest
```

Comment:
Make manually loading in the local CSV a bit easier to consume by not using the local build (which would require some setup to make exist).